channel-jira-server/src/test/java/com/blackduck/integration/alert/channel/jira/server/action/JiraServerGlobalCrudActionsTest.java

```diff
@@ -57,14 +64,20 @@ class JiraServerGlobalCrudActionsTest {
         () -> new PermissionMatrixModel(permissions)
     );
 
+    private final TaskManager taskManager = new TaskManager();
+    private final JiraSchedulingManager jiraSchedulingManager = new JiraSchedulingManager(taskManager);
+    private final TaskScheduler taskScheduler = Mockito.mock(TaskScheduler .class);
```

Comment:
Does the space between TaskScheduler and .class matter?

Reply:
no